src/sql/parts/grid/views/query/chartViewer.component.html

```diff
@@ -150,7 +150,8 @@
 -->
 <ng-template #legendInput>
 	<div class="dialog-label" id="legendLabel">{{localizedStrings.LEGEND}}</div>
-	<select-box class="input-divider" aria-labelledby="legendLabel"
```

Comment:
>aria-labelledby [](start = 35, length = 15)

aria-labelledby doesn't work?

Reply:
The label has to be one level deeper than this. So the aria label is now plumbed through

Reply:
got it! Thanks

---
In reply to: [210669952](https://github.com/Microsoft/sqlopsstudio/pull/2240#discussion_r210669952) [](ancestors = 210669952)

Reply:
I don't care either way really